package.json

```diff
@@ -16,7 +16,7 @@
   "source": "src/index.ts",
   "bin": "bin/vue-i18n-extract.js",
   "main": "dist/vue-i18n-extract.umd.js",
-  "module": "dist/vue-i18n-extract.modern.js",
+  "module": "dist/vue-i18n-extract.modern.mjs",
```

Comment:
Corrected the module field in `package.json` from .js to .mjs to match the actual file in the dist folder. The original extension was probably a typo as no .js file exists in dist, only .mjs.